### ISSUE TITLE
[DataCap Application] Column Sonar Data Archive on AWS 

### ISSUE THREAD

**Issue by lixue-one** [joshua-ne/FIL_DC_Allocator_1022#57]
### Version

1

### DataCap Applicant

Column Sonar Data Archive on AWS

### Project ID

CSDA-002

### Data Owner Name

Column Sonar Data Archive on AWS

### Data Owner Country/Region

United States

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.ncei.noaa.gov/maps/water-column-sonar/; https://cires.gitbook.io/ncei-wcsd-archive

### Social Media Handle

https://www.ncei.noaa.gov/maps/water-column-sonar/; https://cires.gitbook.io/ncei-wcsd-archive

### Social Media Type

Slack

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

10PiB

### Expected size of single dataset (one copy)

2PiB

### Number of replicas to store

5

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f16wp5agwmggn5r23fqf5tq7xxu3hx6dgem7tnjry

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
NOAA collects and uses active acoustic (or sonar) data for a variety of mapping requirements. Water column sonar data focus on the area from near the surface of the ocean to the seafloor. Primary uses of these specific sonar data include 3-D mapping of fish schools and other mid-water marine organisms; assessing biological abundance; species identification; and habitat characterization. Other uses include mapping underwater gas seeps and remotely monitoring undersea oil spills. NCEI archives water column sonar data collected by NOAA line offices, academia, industry, and international institutions. Use the CruisePack data packaging tool to submit water column sonar data to the archive.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Water column sonar data focus on the area from near the surface of the ocean to the seafloor. Primary uses of these specific sonar data include 3-D mapping of fish schools and other mid-water marine organisms; assessing biological abundance; species identification; and habitat characterization. Other uses include mapping underwater gas seeps and remotely monitoring undersea oil spills. NCEI archives water column sonar data collected by NOAA line offices, academia, industry, and international institutions. Use the CruisePack data packaging tool to submit water column sonar data to the archive.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

None

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
About preparing data, we have learned and tried to use lotus code from Filecoin network.
```

### Please share a sample of the data

```text
aws s3 ls --no-sign-request s3://noaa-wcsd-pds/

S3 Bucket: "noaa-wcsd-pds"
├── data
│   ├── processed
│   │   ├── SH1305
│   │   │   ├── 18kHz
│   │   │   │   ├── SaKe_2013-D20130522-T134850.csv
│   │   │   │   ├── SaKe_2013-D20130522-T140446_to_SaKe2013-D20130522-T145239.csv
│   │   │   │   ├── ...
│   │   │   ├── 38kHz
│   │   │   │   ├── ...
│   │   │   ├── 70kHz
│   │   │   │   ├── ...
│   │   │   ├── 120kHz
│   │   │   │   ├── ...
│   │   │   ├── 200kHz
│   │   │   │   ├── ...
│   │   │   ├── bottom
│   │   │   │   ├── SaKe_2013-D20130522-T134850.csv
│   │   │   │   ├── SaKe_2013-D20130522-T140446_to_SaKe2013-D20130522-T145239.csv
│   │   │   │   ├── ...
│   │   │   ├── multifrequency
│   │   │   │   ├── SaKe_2013-D20130522-T134850.csv
│   │   │   │   ├── SaKe_2013-D20130522-T140446_to_SaKe2013-D20130522-T145239.csv
│   │   │   │   ├── ...
│   │   │   ├── ...
│   │   ├── GU1002
│   │   │   ├── ...
│   │   ├── AL0502
│   │   │   ├── ...
│   │   ├── ...
│   ├── raw
│   │   ├── Bell_M_Shimada
│   │   │   ├── SH1305
│   │   │   │   ├── EK60
│   │   │   │   │   ├── SaKe_2013-D20130623-T063450.raw
│   │   │   │   │   ├── SaKe_2013-D20130623-T064452.raw
│   │   │   │   │   ├── SaKe_2013-D20130623-T064452.bot
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, Europe

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), IPFS, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f02984331 --Australia 
f02883857 -- Singapore 
f02852273 -- United Kingdom 
f02973061 -- Russia
f02889193---Vietnam
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by lixue-one** [comment]
@joshua-ne Any updates here?

**Comment by joshua-ne** [comment]
Hi, @lixue-one ， thank you for your application. Most part of the application looks good to me, and for official record, I will post our offline work and my further questions here. 

1. We have done SP localization verification offline, as well as sample data check. Also we have done cross-reference to check the owner/worker/controller addresses of the SPs to make sure not all of the SPs are related to each other. 

2. I can see that the sample data is from NOAA aws, so will the SPs directly download data from there? Or you will preprocess them first and then transfer to SPs

3. In terms of question "How do you plan to make deals to your storage providers", you listed boost client and lotus client. However, I think lotus client is already deprecated. Please confirm or remove this.

**Comment by lixue-one** [comment]
The data is downloaded by us and transmitted to the SP by express delivery. each SP has different technical capabilities, different bandwidths, and different download data, in order to keep the data backup at a reasonable level, we decided to use the hard disk mailing method.we had remove lotus client. 
We are looking forward to seeing your further support for us. We are ready. All SPs support spark and the retrieval rate should be around 75%.

**Comment by joshua-ne** [comment]
Hi everything looks good on my end so far. So let's kick off with a small batch of DC first and see how will things go. Happy onboarding!

**Comment by lixue-one** [comment]
thanks

**Comment by joshua-ne** [comment]
checker:manualTrigger

**Comment by joshua-ne** [comment]
trigger:run_retrieval_test method=lassie sp_list=f02984331,f02883857,f02852273,f02973061,f02889193 client=f16wp5agwmggn5r23fqf5tq7xxu3hx6dgem7tnjry limit=10

**Comment by myfil512** [comment]
| miner_id | retrieval_rate | retrieval_success_counts | retrieval_fail_counts |
| --- | --- | --- | --- |
| f02984331 | NA | 0 | 0 |
| f02883857 | NA | 0 | 0 |
| f02852273 | NA | 0 | 0 |
| f02973061 | NA | 0 | 0 |
| f02889193 | NA | 0 | 0 |

**Comment by joshua-ne** [comment]
Hi @lixue-one  it seems that NONE of the SPs you have sent deals to matches the ones you claimed in the application form. Please explain.

**Comment by joshua-ne** [comment]
trigger:run_retrieval_test method=lassie sp_list=f02865213,f02887063,f02953218 client=f16wp5agwmggn5r23fqf5tq7xxu3hx6dgem7tnjry limit=10

**Comment by myfil512** [comment]
| miner_id | retrieval_rate | retrieval_success_counts | retrieval_fail_counts |
| --- | --- | --- | --- |
| f02865213 | 100% | 10 | 0 |
| f02887063 | 100% | 10 | 0 |
| f02953218 | 100% | 10 | 0 |

**Comment by lixue-one** [comment]
Dear allocator, due to changes in customer pledge, we have changed the cooperating SP. Next, we will not add other SPs. If there are changes in SP, we will disclose it in advance, thank you.

**Comment by joshua-ne** [comment]
Okay, since the SPs support retrieval well, I will continue to another round of support

**Comment by joshua-ne** [comment]
checker:manualTrigger

**Comment by joshua-ne** [comment]
trigger:run_retrieval_test method=lassie sp_list=f02865213,f02887063,f02953218 client=f16wp5agwmggn5r23fqf5tq7xxu3hx6dgem7tnjry limit=10

**Comment by myfil512** [comment]
| miner_id | retrieval_rate | retrieval_success_counts | retrieval_fail_counts |
| --- | --- | --- | --- |
| f02865213 | 90% | 9 | 1 |
| f02887063 | 100% | 10 | 0 |
| f02953218 | 100% | 10 | 0 |

**Comment by joshua-ne** [comment]
The official bot has not been able to pick up the retrieval rate of your SPs yet. But our own check-bot shows good retrieval rates on all 3 SPs. 

However, in your application, you claimed to store 5 duplicates, while currently you have only onboarded 3. Are we expecting more SPs to join any soon? @lixue-one

**Comment by lixue-one** [comment]
Of course. Since the quotas for the first and second rounds were too small, only 50+100, we have only sent 3 SPs . next round, we will add a new SPf02927642 Vietnam. We should have sent an order of 1TiB to it. Maybe the system has not been updated yet. You will see more updates in the next round.

**Comment by lixue-one** [comment]
thanks

**Comment by joshua-ne** [comment]
Sounds fair, I will sign another round now.

**Comment by lixue-one** [comment]
checker:manualTrigger

**Comment by joshua-ne** [comment]
Looks nice to me. Will support another round

**Comment by joshua-ne** [comment]
trigger:run_retrieval_test method=lassie sp_list=f02865213,f02887063,f02927642,f02953218 client=f16wp5agwmggn5r23fqf5tq7xxu3hx6dgem7tnjry limit=10

**Comment by myfil512** [comment]
| miner_id | retrieval_rate | retrieval_success_counts | retrieval_fail_counts |
| --- | --- | --- | --- |
| f02865213 | 100% | 10 | 0 |
| f02887063 | 100% | 10 | 0 |
| f02927642 | 100% | 10 | 0 |
| f02953218 | 100% | 10 | 0 |

**Comment by joshua-ne** [comment]
Retrieval looks good, will support another round.

**Comment by joshua-ne** [comment]
checker:manualTrigger

**Comment by joshua-ne** [comment]
Hi, I see that you claim to store 5 replicas, but only 4 SPs are provided and onboarded. Are we expecting more SPs any soon?

**Comment by lixue-one** [comment]
Yes, dear, we found other SPs. If you can support us again, you will definitely see more progress. Thank you.

**Comment by lixue-one** [comment]
We want to add f03275720,  we are ready, can you support us？

**Comment by lixue-one** [comment]
we do it.

**Comment by lixue-one** [comment]
checker:manualTrigger

**Comment by lixue-one** [comment]
Next, we will add 4 more sps and request more support:
f03512458 HK
f03512720 HK
f03513151 US
f03513182 SG

**Comment by lixue-one** [comment]
checker:manualTrigger

**Comment by lixue-one** [comment]
It looks like the data has not been updated yet.

**Comment by lixue-one** [comment]
A lot of the data is wrong

**Comment by lixue-one** [comment]
@joshua-ne The data is not updating. Is there anything we can do?
Requesting for the next round of support, thanks

**Comment by lixue-one** [comment]
All SPs  support retrieval

**Comment by lixue-one** [comment]
Since the SP we initially cooperated with is no longer cooperating, in order to ensure that the data has at least 5+ backups, we have added 5 new SPs. Thank you.

**Comment by joshua-ne** [comment]
trigger:run_retrieval_test method=lassie sp_list=f02865213,f02887063,f02927642,f02953218,f03315878,f03325608,f03528662,f03528925,f03528938,f03528979,f03529412 client=f16wp5agwmggn5r23fqf5tq7xxu3hx6dgem7tnjry limit=10 start_datetime="2024-08-01 00:00:00" end_datetime="2025-05-06 00:00:00"

**Comment by myfil512** [comment]
| miner_id | retrieval_rate | retrieval_success_counts | retrieval_fail_counts |
| --- | --- | --- | --- |
| f02865213 | 100% | 10 | 0 |
| f02887063 | 100% | 10 | 0 |
| f02927642 | 50% | 5 | 5 |
| f02953218 | 90% | 9 | 1 |
| f03315878 | 0% | 0 | 10 |
| f03325608 | 100% | 10 | 0 |
| f03528662 | 0% | 0 | 10 |
| f03528925 | 0% | 0 | 10 |
| f03528938 | 0% | 0 | 10 |
| f03528979 | 50% | 5 | 5 |
| f03529412 | 10% | 1 | 9 |

**Comment by joshua-ne** [comment]
Hi @lixue-one, it looks like both the retrieval and distribution are not good. We will pause assigning you more data cap until those issues are resolved. You should show us that there is at least one public way to retrieve the data. And PLEASE, do inform us before you change the SP's you are going to work with, not after. This is important.

**Comment by lixue-one** [comment]
checker:manualTrigger

**Comment by joshua-ne** [comment]
hi @lixue-one 

As part of post-review reconciliation, we’re documenting the basis for earlier discontinuation of cooperation, but we also want to ensure that any unresolved or unclear issues are fully reviewed. A few specific points:
The requested DataCap size doesn’t align with the declared dataset size multiplied by the number of intended copies. Can you share the calculation or logic behind your original request?
Actual redundancy was lower than declared — let us know if this was due to resource constraints, client-side changes, or other factors.
Lastly, there’s still no available method for public or sample retrieval. Were you planning to provide any form of access or index (even limited-scope)?
We understand some of these issues may have accumulated gradually, but feel free to share your side so we can keep records clear and fair.
Thanks.
